packages/extension/src/serviceWorker/createCookieObject.ts

```diff
@@ -47,80 +40,60 @@ export async function createCookieObject(
   const value = parsedCookie.value;
 
   const cdpCookie = cookiesList?.find((cookie: Protocol.Network.Cookie) => {
-    return cookie.name === name && cookie.value === value;
+    return cookie.name === name;
```

Comment:
Nope because this gets request from single URL only and if a single URL has same domain and path then the cookie is going to be overridden in URL itself.